ui/shared/PushHealthStatus.jsx

```diff
@@ -99,8 +99,6 @@ class PushHealthStatus extends Component {
               {healthStatus}
             </Badge>
           </a>
-        ) : (
-          <Spinner size="sm" />
```

Comment:
Am I reading this right that the solution to having the infinite spin on the Jobs view is to remove the spinner? :)

Reply:
yup :) Its not necessary for when we have the data and when we don't have the data it just won't render this component.